### PR TITLE
#62 queries execution

### PR DESCRIPTION
**Pull request by kortov**
#62

### CODE REVIEW

File: src/main/resources/sql/select2-1-select2-5.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+-- б) Найти имена покупателей, фамилия которых оканчивается на 'OV'
+SELECT
+  name,
+  surname
+FROM customers
+WHERE surname ILIKE '%OV';
+
+-- в) Найти название самого дорогого по закупке товара, в имени которого присутствует буква 'V',
+--    но не первая и не последняя.
+SELECT
```
Comment (ivshapovalov) [inline src/main/resources/sql/select2-1-select2-5.sql:17]: Please execute query with this values

INSERT INTO
  products (name, purchase_price)
VALUES
  ('vproduct_name', 32.2)
  , ('productv_name', 92.2)
  , ('productv_name', 92.2)
  , ('vproductv_name', 92.2)
  , ('prodVuct_name', 22.2)
  , ('product_namev', 103.2)
  , ('vprovduct_namev', 32.2)
  , ('VVproduct_name', 42.2)
  , ('vproduct_name', 92.2)
  , ('product_namev', 92.2)
  , ('product_name', 92.2)
  , ('prodVVuct_name', 2.2);

Comment (ivshapovalov) [reply src/main/resources/sql/select2-1-select2-5.sql:17]: EXTERNAL query is wrong. Some values, that not satisfy condition can have such price

File: src/main/resources/sql/select2-1-select2-5.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                                FROM products
+                                WHERE name SIMILAR TO '[^Vv]+[Vv]+[^Vv]+'));
+
+
+-- г) Найти имена покупателей, в имени которых присутствует буква 'V' и не больше двух раз
+SELECT
+  name,
+  surname
+FROM customers
+WHERE name SIMILAR TO '([^Vv]*[Vv][^Vv]*){0,2}';
```
Comment (ivshapovalov) [inline src/main/resources/sql/select2-1-select2-5.sql:34]: GOOD job

**Review by ivshapovalov — CHANGES_REQUESTED** [review]

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 1 above)
Comment (kortov) [reply src/main/resources/sql/select2-1-select2-5.sql:17]: Yep, thank you

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 2 above)
Comment (kortov) [reply src/main/resources/sql/select2-1-select2-5.sql:34]: Tnx, I'm surprised that everybody likes my regex))

**Review by Shtramak — APPROVED** [review]
My respect to your regex ;)

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 1 above)
Comment (kortov) [reply src/main/resources/sql/select2-1-select2-5.sql:17]: fixed

**Review by ivshapovalov — APPROVED** [review]

**Review by AVM82 — APPROVED** [review]
Awesome

**Pull request merged**